ihatemoney/default_settings.py

```diff
@@ -9,7 +9,8 @@
 ADMIN_PASSWORD = ""
 ALLOW_PUBLIC_PROJECT_CREATION = True
 ACTIVATE_ADMIN_DASHBOARD = False
-SESSION_COOKIE_SECURE = True
+SESSION_COOKIE_SECURE = False
```

Comment:
```suggestion
SESSION_COOKIE_SECURE = True
```
Don't modify default settings, especially regarding security.